### PR TITLE
updated special case subsection

### PR DESCRIPTION
**Pull request by aecary**

### CODE REVIEW

File: src/section-solving-system-equations-by-graphing.mbx [hunk 1]
```diff
@@ -317,76 +317,115 @@
...
-        </md></p>
-
-        <p>It turned out that those two equations are equivalent. Since they are the same line (or they overlap each other), any point on them is an intersection. We say the system has infinitely many solutions.</p>
-
-        <p>Let's summarize our learning:<ul>
-            <li><p>Parallel Lines: If linear equations in a system have the same slope with different <m>y</m>-intercept, the system has no solution.</p></li>
-            <li><p>Overlapping Lines: If linear equations in a system have the same slope and the same <m>y</m>-intercept, the system has infinitely many solutions.</p></li>
-            <li><p>Intersecting Lines: If linear equations in a system have different slopes, the system has one solution.</p></li>
+        <example xml:id="example-solving-systems-by-grahping-equivalent-equations">
+            <p>Next we'll look at a slightly different special case. Let's start with this system of equations:
```
Comment (Alex-Jordan) [inline src/section-solving-system-equations-by-graphing.mbx]: Here is a `p` that has a `figure` inside it, which is a DTD violation. `figure` is a peer of `p`. I think I would structure this as `p`, `p`, `figure`, `p`.

File: src/section-solving-system-equations-by-graphing.mbx [hunk 2]
```diff
@@ -317,76 +317,115 @@
...
+                    <mrow>6x-3y\subtractright{6x}\amp=12\subtractright{6x}</mrow>
+                    <mrow>-3y\amp=-6x+12</mrow>
+                    <mrow>\multiplyleft{-\frac{1}{3}}(-3y)\amp=\multiplyleft{-\frac{1}{3}}(-6x+12)</mrow>
+                    <mrow>y\amp=2x-4</mrow>
+                </md>
+            As we can see, this is the exact same equation as the other equation in our system. This shows that these two equations are equivalent equations. Any time we try to solve a system of equations where the equations are equivalent, we'll have an infinite set of solutions.
+        </p></remark>
+
+
+        <p>Let's summarize the possible solutions we can have to a system of linear equations:<ul>
```
Comment (Alex-Jordan) [inline src/section-solving-system-equations-by-graphing.mbx]: Would this `ul` be better as a `dl`? Things like "Parallel lines:" always raise this question in my mind.

**Comment by aecary** [comment]
Ok, I'll change that and watch for it. I might have quite a few of those.

Ann Cary
Mathematics Instructor
Portland Community College
Sylvania Campus, ST 203
971-722-7590
ann.cary@pcc.edu
<http://goog_233805033>
http://spot.pcc.edu/~acary/
Pronouns: she/her/hers

**Comment by aecary** [comment]
Yes, that crossed my mind and I was wondering the same thing. I'll change
that.

Ann Cary
Mathematics Instructor
Portland Community College
Sylvania Campus, ST 203
971-722-7590
ann.cary@pcc.edu
<http://goog_233805033>
http://spot.pcc.edu/~acary/
Pronouns: she/her/hers

**Pull request merged**